ProjectSettings/TagManager.asset

```diff
@@ -3,8 +3,7 @@
 --- !u!78 &1
 TagManager:
   serializedVersion: 2
-  tags:
-  - Card
+  tags: []
```

Comment:
May need to ask @j-e-b what this tag was, and if it's an issue that it was removed

Reply:
This must be some kind of merging conflict. I didn't touch any tags.